locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,57 @@
+# links
+-link-firefox-password-manager = <a href="https://www.mozilla.org/en-US/firefox/features/password-manager/">Firefox Password Manager</a>
+-link-mozilla-vpn = <a href="https://www.mozilla.org/en-US/products/vpn/">Mozilla VPN</a>
+-link-firefox-relay = <a href="https://relay.firefox.com/">Firefox Relay</a>
+-link-equifax = <a href="https://www.equifax.com/personal/credit-report-services/credit-freeze/">Equifax</a>
+-link-experian = <a href="https://www.experian.com/freeze/center.html">Experian</a>
+-link-transunion = <a href="https://www.transunion.com/credit-freeze">TransUnion</a>
+
+# password
+breach-checklist-pw-header = Go to {$companyLink} to change your password and enable two-factor authentication (2FA). 
+breach-checklist-pw-body = Make sure your password is unique and hard to guess. If this password is used on any other accounts, be sure to change it there too. {-link-firefox-password-manager} can help you securely keep track of all of your passwords.
+
+# email
+breach-checklist-email-header = Protect your email with an email masking service like {-link-firefox-relay}.
+breach-checklist-email-body = This can hide your true email address while forwarding emails to your real inbox.
+
+# social security number
+breach-checklist-ssn-header = Monitor your credit report for accounts, loans, or credit cards you don't recognize.
+breach-checklist-ssn-body = You can also consider freezing your credit on {-link-equifax}, {-link-experian} and {-link-transunion} to stop scammers from opening new accounts in your name. It’s free and won’t affect your credit score.
```

Comment:
This group needs a clear explanation, as this is something that makes sense only for US citizens.

I wouldn't define terms for external brands.

Reply:
Looping in content/copy for an appropriate explanation here.